symfony/asset-mapper/6.4/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "copy-from-recipe": {
+        "assets/": "assets/",
+        "config/": "%CONFIG_DIR%/",
+        "importmap.php": "importmap.php"
+    },
+    "aliases": ["asset-mapper", "importmap"],
+    "gitignore": [
+        "/%PUBLIC_DIR%/assets/",
+        "/assets/vendor"
+    ],
+    "add-lines": [
+        {
+            "file": "templates/base.html.twig",
+            "content": "            {{ importmap('app') }}",
+            "position": "after_target",
+            "target": "{% block javascripts %}",
+            "warn_if_missing": true
+        }
+    ],
+    "conflict": {
+        "symfony/framework-bundle": "<6.4",
+        "symfony/twig-bundle": "<6.4",
```

Comment:
These 2 lines can stay at `<6.3` like the original version. AssetMapper 6.4 requires FW Bundle 6.4, but that is handled by composer in the normal way. These lines were here just to make sure that the user had at least 6.3 of both of those bundles because that is what powers the `{{ importmap() }}` function. From the recipes point of view, 6.3 is still enough.